pages/rdm_cycle/reusing.md

```diff
@@ -1,37 +1,45 @@
 ---
 title: Reusing
-keywords:
-contributors: [Korbinian Bösl, Daniel Faria]
+keywords: [Data reuse, Reusable data, Licencing, Licence, Data integration, Versioning, Citing data, Data citation, FAIR]
```

Comment:
I suggest to keep only "reuse data" and "licence" and "data integration" and "citing data". Redundant tags are not needed because suggested pages appear in the search box meanwhile the user is still typing. The search box already looks for part of the word, so perfect match is not needed.

FAIR and Versioning seems a bit too far for this page, in my opinion.

Reply:
Sounds reasonable to remove the redundant keywords! I didn't really know what to put in, since so many pages were still lacking keywords.

Reply:
I also think it is fine to remove "FAIR" and "Versioning"

Reply:
I have updated the keywords according to your suggestion.